src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -149,11 +149,41 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_EXPAND = 'Expand';
     const STRING_COLLAPSE = 'Collapse';
+    const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
+    const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
+    const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
+    const isCollapsePanelsButton = !panels;
+    /*
+     * When collapsing expandCollapseButton, all panels buttons under it,
+     * (expandCollapseTeamButton and  expandCollapseStudentButton) should be in collapsed state,
+     * so that next time we expand the main panel, we find the expandCollapseTeamButton and
+     * expandCollapseStudentButton in collapsed state.
+     */
+    if (isCollapsePanelsButton && !isButtonInExpandMode) {
+        if ($(expandCollapseStudentsButton).length
+            && $(expandCollapseStudentsButton).html().trim().startsWith(STRING_EXPAND)) {
+            replaceButtonHtmlAndTooltipText(expandCollapseStudentsButton, STRING_EXPAND, STRING_COLLAPSE);
+            $(expandCollapseStudentsButton).trigger('mouseleave');
```

Comment:
Does the tooltip still show up using the new approach of `replaceButtonHtmlAndTooltipText`?

Reply:
@whipermr5 Yes, because we show tooltip after replacing text.
https://github.com/TEAMMATES/teammates/blob/cdcc5c8e81db97c5ff5146c248ac0de2cabe461f/src/main/webapp/dev/js/common/instructorFeedbackResults.js#L139-L141

Reply:
Then move that logic out of `replaceButtonHtmlAndTooltipText` to where the method is currently used.

Reply:
Then move that logic out of `replaceButtonHtmlAndTooltipText` to where the method is currently used.